graylog2-web-interface/src/components/lookup-tables/LookupTableView.tsx

```diff
@@ -0,0 +1,175 @@
+      });
+    }
+  };
+
+  return (
+    <Row className="content">
+      <Col md={6}>
+        <h2>{table.title}</h2>
+        <p>{table.description}</p>
+        <dl>
+          <dt>Data adapter</dt>
+          <dd>
+            <Link to={Routes.SYSTEM.LOOKUPTABLES.DATA_ADAPTERS.show(dataAdapter.name)}>
+              {dataAdapter.title}
+            </Link>
+          </dd>
+          <dt>Cache</dt>
+          <dd>
+            <Link to={Routes.SYSTEM.LOOKUPTABLES.CACHES.show(cache.name)}>{cache.title}</Link>
+          </dd>
+        </dl>
+        {(!loadingScopePermissions && scopePermissions?.is_mutable) && (
+          <Button bsStyle="success"
+                  onClick={handleEdit(table.name)}
+                  role="button"
+                  name="edit">
+            Edit
+          </Button>
+        )}
+        {(table.default_single_value || table.default_multi_value) && (
+          <dl>
+            <dt>Default single value</dt>
+            <dd><code>{table.default_single_value}</code>{' '}({table.default_single_value_type.toLowerCase()})</dd>
+            <dt>Default multi value</dt>
+            <dd><code>{table.default_multi_value}</code>{' '}({table.default_multi_value_type.toLowerCase()})</dd>
+          </dl>
+        )}
+        <hr />
+        <h2>Purge Cache</h2>
+        <p>You can purge the complete cache for this lookup table or only the cache entry for a single key.</p>
+        <form onSubmit={handlePurgeKey}>
+          <fieldset>
+            <Input type="text"
+                   id="purge-key"
+                   name="purgekey"
+                   label="Key"
+                   onChange={handleInputOnChange}
+                   help="Key to purge from cache"
+                   required
+                   value={purgeKey.value} />
+            <ButtonToolbar>
+              <Button type="submit" bsStyle="success" disabled={!purgeKey.valid}>Purge key</Button>
+              <Button type="button" bsStyle="info" onClick={hadlePurgeAll}>Purge all</Button>
+            </ButtonToolbar>
+          </fieldset>
+        </form>
+      </Col>
+      <Col md={6}>
+        <h2>Test lookup</h2>
+        <p>You can manually query the lookup table using this form. The data will be cached as configured by Graylog.</p>
+        <form onSubmit={handleLookupKey}>
+          <fieldset>
+            <Input type="text"
+                   id="key"
+                   name="lookupkey"
+                   label="Key"
+                   required
+                   onChange={handleInputOnChange}
+                   help="Key to look up a value for."
+                   value={lookupKey.value} />
+            <Button type="submit" bsStyle="success" disabled={!lookupKey.valid}>Look up</Button>
+          </fieldset>
+        </form>
+        {lookupResult && (
+          <div style={{ marginTop: '16px' }}>
+            <h4>Lookup result</h4>
+            <pre>{JSON.stringify(lookupResult, null, 2)}</pre>
```

Comment:
Maybe let's memorise the result of `JSON.stringify`?

Reply:
I think this a good call. We can do that once we move to react-query. Right now this value is stored in a state since is the result of a query request where the user can input a key_name to get some data back.